### PR TITLE
Update chapter_one.txt

### PR DESCRIPTION
**Pull request by alvinjchitena**
@mkphuthi @BukhosiMoyo For Intro to Computers, I replaced the placeholder text with actual content. I also added some connective tissue, so that the concepts flow from one to other, eventually leading up to programming. 

I was brief in this section, because our main focus is programming. Please add any key concepts I may have missed, especially on the use of the command line.

### CODE REVIEW

File: 12-week Course/2. Chapter One/chapter_one.txt [hunk 1]
```diff
@@ -20,39 +21,88 @@ turning off the lights.
...
+3. Close a program
+4. Delete a file
+5. Open a webpage in a browser
+6. Do arithmetic
+7. Shut down a computer
+
+Here are some basic command line commands:
+
+help - line lists all the basic builtin commands at your disposal.
+dir - list all the contents inside the current directory
```
Comment (alvinjchitena) [inline 12-week Course/2. Chapter One/chapter_one.txt]: @mkphuthi your material uses the unix version - ls. The exercises will need to be edited to use the dos version.

File: 12-week Course/2. Chapter One/chapter_one.txt [hunk 2]
```diff
@@ -20,39 +21,88 @@ turning off the lights.
...
+e.	make a new folder called “MyText” using mkdir
+f.	cd into the folder
+g.	Use the command notepad myfile.txt to create a new text file and save it.
+h.	Use the GUI browser (Windows Explorer) to see if the file is where you put it.
+
+Exercise: 1.y: Genius Hacker!
 
-{ CODE DEMONSTRATION: 
-  Fundamental CMD commands + arithmetic in CMD demo }
+Try to figure out how to open the Zim Code website (zimcode.org) in the Google Chrome browser, using only the command
```
Comment (alvinjchitena) [inline 12-week Course/2. Chapter One/chapter_one.txt]: The VLC example should give the clever ones an idea of how to do this.

File: 12-week Course/2. Chapter One/chapter_one.txt [hunk 3]
```diff
@@ -1,8 +1,9 @@
-Chapter 1 – Introduction to Python
+Chapter 1 – Introduction to Computers & Python
```
Comment (alvinjchitena) [inline 12-week Course/2. Chapter One/chapter_one.txt]: Changed title to include Computers intro.

File: 12-week Course/0. E Quizes/Quiz 1.txt [hunk 4]
```diff
@@ -0,0 +1,186 @@
+Name:						 
+Group:
+Computer Number:
```
Comment (alvinjchitena) [inline 12-week Course/0. E Quizes/Quiz 1.txt]: Stylistic change. Vertical listing to make it easier to read.

File: 12-week Course/0. E Quizes/Quiz 1.txt [hunk 5]
```diff
@@ -0,0 +1,186 @@
+Name:						 
+Group:
+Computer Number: 				
+
+Quiz 1 (1hr 45 minutes) [Exam Total: 46 marks]
```
Comment (alvinjchitena) [inline 12-week Course/0. E Quizes/Quiz 1.txt]: Adding to time and marks for clarity.

File: 12-week Course/0. E Quizes/Quiz 1.txt [hunk 6]
```diff
@@ -0,0 +1,186 @@
...
+
+Do not panic! We are using this test to see how you guys are doing. It’s not a midyear or exam. You are allowed to use notes and ask for hints for some of the questions. Please read all instructions carefully before attempting questions. Remember to write you name, group and computer number at the top.
+
+Part A: Short Questions [Total: 30 marks]
+
+You are advised to spend x minutes on this section.
```
Comment (alvinjchitena) [inline 12-week Course/0. E Quizes/Quiz 1.txt]: Advise for students to pace themselves.

File: 12-week Course/0. E Quizes/Quiz 1.txt [hunk 7]
```diff
@@ -0,0 +1,186 @@
...
+
+Boolean (True/False) Questions [5 marks] {ET: 4mins}
+	2 different files can have the same file path on the same hard drive. [1]
+	Python is the only programming language. [1]
+	A script is used to write short commands while the shell is used to write longer programs. [1]
+	You must always import the math module before you use it. [1]
+	A function can have no parameters. [1]
+{
+
+ Multiple Choice [5 marks] {ET: 4mins}
```
Comment (alvinjchitena) [inline 12-week Course/0. E Quizes/Quiz 1.txt]: This one multiple choice question is awkward. See suggestion below it in { }.

File: 12-week Course/0. E Quizes/Quiz 1.txt [hunk 8]
```diff
@@ -0,0 +1,186 @@
...
+def myfunc(x):
+return x + 2
+
+#5
+s = “py”
+y = s[2]
+
+
+Why does this not print anything in the script? [1] 
+{ This should be separate from the other five, since it has its own question }
```
Comment (alvinjchitena) [inline 12-week Course/0. E Quizes/Quiz 1.txt]: This has a separate question, so should be separate from the code block stuff.

File: 12-week Course/0. E Quizes/Quiz 1.txt [hunk 9]
```diff
@@ -0,0 +1,186 @@
...
+Babes WODUMO
+Your name has 11 characters 
+
+{ Total exam time + 10mins for instructions & delays = ~1 hour 
+  Consider:
+  1. Adding a few more small concept questions, multiple choice/TorF and code block questions {ET: 15 mins}
+  2. Adding a second long script problem.
+  3. Making the coding questions a bit harder by adding a few more concepts and steps {ET 30 mins}
+  Making the exam length total ~ 1 hour 45 mins. They can get 10 extra minutes.
+}
```
Comment (alvinjchitena) [inline 12-week Course/0. E Quizes/Quiz 1.txt]: Suggestions to improve exam quality.

File: 12-week Course/2. Chapter One/chapter_one.txt (hunk 1 above)
Comment (mkphuthi) [reply 12-week Course/2. Chapter One/chapter_one.txt]: ls works in newer versions of dos. I think at the time I wanted to use commands that are as universal as possible but right now I htink consistency is best so let's do dir

**Pull request merged**